company.el

```diff
@@ -2912,7 +2919,7 @@ Delay is determined by `company-tooltip-idle-delay'."
 (defun company-preview-show-at-point (pos)
   (company-preview-hide)
 
-  (let ((completion (nth company-selection company-candidates)))
+  (let ((completion (if (cdr company-candidates) company-common (nth company-selection company-candidates))))
```

Comment:
Thank you for the good advice. I will change the function to take a new argument.